src/test/java/orionmipt/LJVTest.java

```diff
@@ -0,0 +1,206 @@
+package orionmipt;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+public class LJVTest {
+
+    private final LJV ljv = new LJV();
+
+
+    @Test
+    void StringIsNotAPrimitiveType() {
```

Comment:
Method names should start with small letter (unlike classes, which start from capital letter)